studio/schemas/resource.js

```diff
@@ -252,6 +230,13 @@ export default {
       type: 'array',
       of: [{type: 'link'}],
     },
+    {
+      name: 'images',
```

Comment:
a stack of labeled image urls so you can collect assets and use the labels to query them